### PR TITLE
Update engagements.md

### PR DESCRIPTION
**Pull request by gstout52**
Revised quote page with new process.

## Description

Added steps to create a new quote in HubSpot.

## Related Issue(s)

N/A

## Checklist

 - [X] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [X] I have considered the performance impact of these changes
 - [] Suitable unit/system level tests have been added and they pass
 - [X] Documentation has been updated

### CODE REVIEW

**Review by zackwasli — APPROVED** [review]
Looks great, not sure why the tests are failing though it looks like? There may be other sections of the handbook that link to this page and changing the title of the subsection could be generating the error? Total guess, sorry I can't be of more help

File: src/handbook/customer/sales/engagements.md [hunk 1]
```diff
@@ -6,12 +6,50 @@ navTitle: Engagements & Pricing
...
+
+EXAMPLE 1:
+
+Suppose that a customer wants to purchase a year of the Team plan with 5 small instances, the minimum number for the Team plan. That quote would look like this (ignoring any discounts or other extras):
+
+| Item & Description | Quantity | Unit Price | Total |
+| ----- | ----- | ----- | ----- |
+| FlowFuse Team Platform \- Cloud | 1 | $X,000/year | $X,000 |
+
+Annual Subtotal	  $2,100
```
Comment (gstout52) [inline src/handbook/customer/sales/engagements.md:25]: Annual Subtotal	  $X,000

Comment (gstout52) [inline src/handbook/customer/sales/engagements.md:25]: Annual Subtotal	  $X,000

File: src/handbook/customer/sales/engagements.md [hunk 2]
```diff
@@ -6,12 +6,50 @@ navTitle: Engagements & Pricing
...
+EXAMPLE 1:
+
+Suppose that a customer wants to purchase a year of the Team plan with 5 small instances, the minimum number for the Team plan. That quote would look like this (ignoring any discounts or other extras):
+
+| Item & Description | Quantity | Unit Price | Total |
+| ----- | ----- | ----- | ----- |
+| FlowFuse Team Platform \- Cloud | 1 | $X,000/year | $X,000 |
+
+Annual Subtotal	  $2,100  
+**Total**			  $2,100
```
Comment (gstout52) [inline src/handbook/customer/sales/engagements.md:26]: **Total**			  $X,000

**Comment by sumitshinde-84** [comment]
It will fail because the current branch doesn’t have the updated billing.md file. Instead of creating another PR, it would have been better to make the changes in the same branch. However, to pass the test, please pull the main branch into your current branch.

**Comment by gstout52** [comment]
> It will fail because the current branch doesn’t have the updated billing.md file. Instead of creating another PR, it would have been better to make the changes in the same branch. However, to pass the test, please pull the main branch into your current branch.

Thank you, @sumitshinde-84 . Can you tell me how to do that?

**Comment by sumitshinde-84** [comment]
Yes, please make sure you are on the `gstout52-patch-2 branch`, then execute `git pull origin main`

**Pull request merged**